cogs/duel.py

```diff
@@ -163,18 +182,22 @@ def indicatize(d):
 HEALS = {
     'inject': {
         'morphine': 4,
-        'nanomachines': 5
+        'nanomachines': 5,
+        'fentanyl': 4
```

Comment:
Not sure if this is a good one to add.  Would be best to leave this out in my opinion.